[Draft] https://issues.redhat.com/browse/ACM-7601 Adding Search APIs (#5609)
open-cluster-management/rhacm-docs: apis/search_operator.json.adoc

[#search-api]
= Search operator API

[[_rhacm-docs_apis_searchoperator_jsonoverview]]
== Overview

> This documentation is for the searchCustomization resource for {product-title}. The SearchCustomization resource has four possible requests: create, query, delete and update. 

Author: ```suggestion

This documentation is for the search operator for {product-title}. The search operator has four possible requests: create, query, delete and update. 
```

=== Version information
[%hardbreaks]
_Version_ : 2.9.0

=== URI scheme
[%hardbreaks]
_BasePath_ : /kubernetes/apis
_Schemes_ : HTTPS

=== Tags

* searches.open-cluster-management.io/v1 : Create and manage the search operator

[[_rhacm-docs_apis_searchoperator_jsonpaths]]
== Paths

[[_rhacm-docs_apis_searchoperator_jsoncreate]]
=== Create a search list
....
POST /searches.open-cluster-management.io/v1/v1alpha1/namespaces/{namespace}/policies/{policy_name}
....

==== Description

> Create a searchList.

Author: ```suggestion
Create a searchList by using the search operator.
```

> [#search-api]

Reviewer: @dockerymick I noticed that you started from the policy API doc.  It will be much simpler if we start from `apis/multicluster_hub.json.adoc` and make updates from there.

[#search-api]
= Search operator API

[[_rhacm-docs_apis_searchoperator_jsonoverview]]
== Overview
This documentation is for the searchCustomization resource for {product-title}. The SearchCustomization resource has four possible requests: create, query, delete and update. 

=== Version information
[%hardbreaks]
__Version__ : 2.9.0

=== URI scheme
[%hardbreaks]
__BasePath__ : /kubernetes/apis
__Schemes__ : Open API

=== Tags

* searches.open-cluster-management.io/v1 : Create and manage the search operator

[[_rhacm-docs_apis_searchoperator_jsonpaths]]
== Paths

[[_rhacm-docs_apis_searchoperator_jsoncreate]]

> === Create a searh list

Reviewer 2: ```suggestion
=== Create a search list
```
